openfasoc/generators/ldo-gen/tools/ldo-gen.py

```diff
@@ -185,54 +185,69 @@
         tuple((prePEX_specialized_run_dir, "power_array.spice", powerArrayNetlist))
     )
 
-    # prepare simulation scripts (return as strings)
+    shutil.copy(
+        directories["simDir"] + "/templates/.spiceinit", prePEX_specialized_run_dir
+    )
+    shutil.copy(
+        directories["simDir"] + "/templates/.spiceinit", postPEX_specialized_run_dir
+    )
+
+    # write all the files to their respective locations
+    for filetocopy in filestocopy:
+        with open(filetocopy[0] + "/" + filetocopy[1], "w") as simfile:
+            simfile.write(filetocopy[2])
+
+    # prepare simulation scripts and run simulations (return as strings)
     if jsonConfig["simTool"] == "ngspice":
-        [prePEXscript, PEXscript, PWRARRscript] = prepare_scripts_ngspice(
+        [prePEXscript, PEXscript, PWRARRscript] = prepare_scripts_and_run_ngspice(
             directories["simDir"] + "/templates/",
+            prePEX_specialized_run_dir,
+            postPEX_specialized_run_dir,
             pdk_path,
+            arrSize,
             "tt",
             user_specs["designName"],
             user_specs["vin"],
+            prePEX=False,
         )
     # elif jsonConfig["simTool"] == "Xyce":
     else:
         print("simtool not supported")
         exit(1)
-    filestocopy.append(
-        tuple((postPEX_specialized_run_dir, "ldoInst_sim.sp", PEXscript))
-    )
-    filestocopy.append(
-        tuple((prePEX_specialized_run_dir, "ldoInst_sim.sp", prePEXscript))
-    )
-    filestocopy.append(
-        tuple(
-            (
-                prePEX_specialized_run_dir,
-                "power_array_template_ngspice.sp",
-                PWRARRscript,
-            )
-        )
-    )
-
-    # write all the files to their respective locations
-    for filetocopy in filestocopy:
-        with open(filetocopy[0] + "/" + filetocopy[1], "w") as simfile:
-            simfile.write(filetocopy[2])
-    shutil.copy(
-        directories["simDir"] + "/templates/.spiceinit", prePEX_specialized_run_dir
-    )
-    shutil.copy(
-        directories["simDir"] + "/templates/.spiceinit", postPEX_specialized_run_dir
-    )
+    # filestocopy.append(
```

Comment:
why is this commented out?

Reply:
Earlier this was used to copy sim files to specific directory. Now the copying of files is done in the function of simulationa. py file

Reply:
Should be deleted then

Reply:
Okay. I will delete it and push a commit. Just give me a minute